runner/dl_checkpoints.sh

```diff
@@ -147,19 +135,25 @@ function build_tensorrt_models() {
                 " \
         || (echo "failed streamdiffusion tensorrt"; return 1)
 
-    # ComfyUI (only DepthAnything for now)
-    AI_RUNNER_COMFYUI_IMAGE=${AI_RUNNER_COMFYUI_IMAGE:-livepeer/ai-runner:live-app-comfyui}
-    docker pull $AI_RUNNER_COMFYUI_IMAGE
-    # ai-worker has tags hardcoded in `var livePipelineToImage` so we need to use the same tag in here:
-    docker image tag $AI_RUNNER_COMFYUI_IMAGE livepeer/ai-runner:live-app-comfyui
+    # Depth-Anything-Tensorrt
+    docker run --rm -v ./models:/models --gpus all -l TensorRT-engines $AI_RUNNER_COMFYUI_IMAGE \
+    bash -c "cd /ComfyUI/models/tensorrt/depth-anything && \
+                python /ComfyUI/custom_nodes/ComfyUI-Depth-Anything-Tensorrt/export_trt.py && \
+                adduser $(id -u -n) && \
+                chown -R $(id -u -n):$(id -g -n) /models" \
+    || (echo "failed ComfyUI Depth-Anything-Tensorrt"; return 1)
+
+    # Dreamshaper-8-Dmd-1kstep
+    # TODO: Remove the script download with curl. It should already come in the base image once eliteprox/comfystream#1 is merged.
     docker run --rm -v ./models:/models --gpus all -l TensorRT-engines $AI_RUNNER_COMFYUI_IMAGE \
-        bash -c "cd /comfyui/models/Depth-Anything-Onnx && \
-                    python /comfyui/custom_nodes/ComfyUI-Depth-Anything-Tensorrt/export_trt.py && \
-                    mkdir -p /comfyui/models/tensorrt/depth-anything && \
-                    mv *.engine /comfyui/models/tensorrt/depth-anything && \
-                    adduser $(id -u -n) && \
-                    chown -R $(id -u -n):$(id -g -n) /models" \
-        || (echo "failed ComfyUI tensorrt"; return 1)
+        bash -c "cd /comfystream/src/comfystream/scripts && \
+                 curl -O https://raw.githubusercontent.com/pschroedl/comfystream/refs/heads/10_29/build_trt/src/comfystream/scripts/build_trt.py && \
```

Comment:
What's the story behind the `pschroedl` fork.
Are there plans to merge it back to `eliteprox`?